### DISCUSSION TITLE
Problem with username and password

### DISCUSSION THREAD

**Discussion by thisisbommy** [icloud-photos-downloader/icloud_photos_downloader#736] [Q&A]
First time trying to use this software (thank you!) on a Windows PC. I've downloaded it and got it running, but it's showing a username/password error. Thing is, the username and password are definitely right. I pasted them from a text file into my icloudpd prompt in my cmd window, and then into a browser logging onto iCloud. The former fails, the latter works straight away.

Any ideas?

<img width="1620" alt="error" src="https://github.com/icloud-photos-downloader/icloud_photos_downloader/assets/147372860/73dfc5ab-b5a0-46f6-9efa-d737ca05fce6">

**Comment by scaraebeus** [comment]
@thisisbommy - see Issue #729. It's a known issue.  There is info on a fix in there, however, it has not been merged into the main branch at this time so is not fully tested, validated, verified.  It appears to be working for some though if you are able to consume it.

**Reply by scrutinizer11** [reply]
So nice to see someone having a similar problem. I opened the issue just now: [https://github.com/icloud-photos-downloader/icloud_photos_downloader/issues/772](https://github.com/icloud-photos-downloader/icloud_photos_downloader/issues/772)
